tests/phpunit/Configuration/ConfigurationFactoryTest.php

```diff
@@ -73,10 +76,26 @@ public static function tearDownAfterClass(): void
 
     protected function setUp(): void
     {
+        /** @var SourceFileCollector&ObjectProphecy $sourceFilesCollectorProphecy */
+        $sourceFilesCollectorProphecy = $this->prophesize(SourceFileCollector::class);
```

Comment:
I used prophecy here because I had no idea how to do it with `Mock`. If someone has the alternative I'll take it, otherwise the following works...